demos/python_demos/speech_recognition_demo/README.md

```diff
@@ -18,16 +18,20 @@ The demo depends on the `ctcdecode_numpy` Python extension module,
 which implements CTC decoding in C++ for faster decoding.
 Please refer to [Using Open Model Zoo demos](../../README.md) for instructions
 on how to build the extension module and prepare the environment for running the demo.
+Alternatively, you can [install ctcdecode-numpy from OpenVINO 2021.1 Open Model Zoo](https://github.com/openvinotoolkit/open_model_zoo/tree/2021.1/demos/python_demos/speech_recognition_demo/ctcdecode-numpy#installation).
```

Comment:
is it possible to use relative path to the current version of file instead of refer link to documentation of two releases in past?
@IRDonch FYI

Reply:
I replaced this sentence. I added the `setup.py` file back, and now user has two options to build : either with cmake, or with "pip install ."  I fear that leaving only the cmake option can scare off those users who are Python developers: they are less familiar with cmake/make, and more familiar with pip/setup.py.  I kept Roman's version of CMakeLists.txt.

Reply:
did you notice [PR](https://github.com/openvinotoolkit/open_model_zoo/pull/1783) from Roman?

Reply:
Yes, i noticed it. That's why I'm changing something here. Did you notice that this change is marked as "Outdated"? I changed it again in the commit below called "Address reviewers' comments".

Roman earlier removed `setup.py` and added `CMakeList.txt` instead. What I'm saying is that it would be better to keep my `setup.py` in addition to Roman's `CMakeList.txt`. Roman didn't like that the old `CMakeList.txt` called pip. Now with the new Roman's `CMakeList.txt`, cmake path doesn't call pip. I'd like to keep pip/`setup.py` path for the users who are interested in checking/using only a single demo (as opposed to building all demos), and for those who are more familiar with Python/pip than with C++/cmake/make.

Making life too hard for the users can easily cause them to use other solutions for speech recognition that work out of the box (but somewhat slower) and can be installed with just a single command.

Reply:
Oh, I just now understood that you were talking about PR #1783, not PR #1766. I didn't notice that it was not merged yet. I've left a comment there now.